templates/partials/taxonomylist.html.twig

```diff
@@ -1,10 +1,13 @@
 {% set taxlist = children_only is defined ? taxonomylist.getChildPagesTags() : taxonomylist.get() %}
+{% set limit = config.plugins.taxonomylist.limit %}
 
 {% if taxlist %}
 <span class="tags">
-    {% for tax,value in taxlist[taxonomy] %}
-        {% set active = uri.param(taxonomy) == tax? 'active' : '' %}
-        <a class="{{ active }}" href="{{ base_url }}/{{ taxonomy }}{{ config.system.param_sep }}{{ tax }}">{{ tax }}</a>
+    {% set count = 0 %}
+    {% for tax,value in taxlist[taxonomy] if count < limit %}
```

Comment:
`count < limit` could cause issues if you have no limit value.  need to also check if limit is set before this if is evaluated